common/web/types/src/kmx/kmx-file-reader.ts

```diff
@@ -59,32 +66,39 @@ export class KmxFileReader {
     result.startGroup = {
       ansi: binaryKeyboard.StartGroup_ANSI == 0xFFFFFFFF ? -1 : binaryKeyboard.StartGroup_ANSI,
       unicode: binaryKeyboard.StartGroup_Unicode == 0xFFFFFFFF ? -1 : binaryKeyboard.StartGroup_Unicode,
-      newContext: -1, //TODO
-      postKeystroke: -1 // TODO
+      newContext: -1,
+      postKeystroke: -1
```

Comment:
... what here got moved from "TODO" to "to-done"?

Reply:
The `TODO` items were done a while ago and I forgot to remove the comments when I fixed them up